apps/web/src/components/layouts/FocusLayout.tsx

```diff
@@ -2,16 +2,19 @@ import { ReactNode } from 'react'
 
 export default function FocusLayout({
   header,
+  footer,
   children,
 }: {
   header: ReactNode
+  footer: ReactNode
   children?: ReactNode
 }) {
   return (
     <div className='flex flex-col items-center justify-center h-screen'>
       <div className='flex flex-col gap-y-6 max-w-[22rem]'>
         <div>{header}</div>
         {children}
+        <div>{footer}</div>
```

Comment:
Do an if here and put that focus footer div styles
```javascript
{footer ? (<div classNmame="...">{footer}</div>) : null}
```

Reply:
Why the styling there? The header is not doing that

Reply:
Good point. Because people change opinions. I did the header and now I changed my opinion.